.ahoy.yml

```diff
@@ -10,7 +10,11 @@ commands:
   docker:
     #cmd: "ahoy help home"
     usage: Example docker commands from examples/docker.ahoy.yml
-    import: examples/docker.ahoy.yml
+    import: |
```

Comment:
If we use an array here instead of a string we will break backwards compatibility with all of the previous ahoy files.